src/likelihoodratiotest.jl

```diff
@@ -166,9 +166,12 @@ function _iscomparable(m::LinearMixedModel...)
     true
 end
 
+_samefamily(::GeneralizedLinearMixedModel{T,S}...) where {T, S<:Distribution} = true
```

Comment:
```suggestion
_samefamily(::GeneralizedLinearMixedModel{T, S}...) where {T, S} = true
```
The subtyping restriction is redundant here since a GLMM object can't have a `S` that isn't a `Distribution`.